src/index.html

```diff
@@ -11,7 +11,208 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+        Personal information
+        </legend>
+        <div>
```

Comment:
Put empty line between close & open tag
use it in all cases
```suggestion
        </legend>
        
        <div>
```